### PR TITLE
Adjust .coveragerc to include all of src

### PR DESCRIPTION
**Pull request by bdraco**
https://github.com/aio-libs/propcache/pull/37/files#r1790561516

### CODE REVIEW

**Comment by bdraco** [comment]
Seems to confuse codecov https://app.codecov.io/gh/aio-libs/propcache/pull/49/tree

File: .coveragerc [hunk 1]
```diff
@@ -4,7 +4,7 @@ skip_covered = false
 
 [paths]
 source =
-  src/propcache
+  src
   */lib/pypy*/site-packages/propcache
   */lib/python*/site-packages/propcache
   */Lib/site-packages/propcache
```
Comment (webknjaz) [inline .coveragerc:10]: This entire section is not inclusion but simply a path mapping definition
```suggestion
  */lib/pypy*/site-packages
  */lib/python*/site-packages
  */Lib/site-packages
```

File: .coveragerc [hunk 2]
```diff
@@ -4,7 +4,7 @@ skip_covered = false
 
 [paths]
 source =
```
Comment (webknjaz) [inline .coveragerc:6]: I was given feedback that this name is confusing (or `src` that I was using in some projects) so I'm now migrating my projects to something a bit more verbose in hopes of preventing people from making assumptions…
```suggestion
_site-packages-to-src-mapping =
```

**Comment by webknjaz** [comment]
> Seems to confuse codecov

My commits should've fixed that. Rebased to verify.

**Pull request merged**